### ISSUE TITLE
[Question] Replacement for lines-of-executable-code

### ISSUE THREAD

**Issue by spydon** [dart-code-checker/dart-code-metrics#317]
**What do you want to discuss?**
Is there a replacement for `lines-of-executable-code` that was deprecated in #291?
We are currently getting a warning about that being deprecated in out pipeline, but we can't remove it either since we increase the default number there so if we remove it we get a warning in the actual analysis.

**Comment by dkrutskikh** [comment]
@spydon use `Source lines of Code`

**Comment by spydon** [comment]
Makes sense, I still get the warning though. I'm probably doing something wrong then?
```
analysis_options.yaml

dart_code_metrics:
   ...
  metrics:
    ...
    source-lines-of-code: 200
```
Result:
`WARNING addControlsStories - lines of executable code: 61`

**Comment by dkrutskikh** [comment]
@spydon we released version 3.2.1 with fix this issue

**Comment by spydon** [comment]
> @spydon we released 3.2.1 version with fix this issue

Nice, thanks for the very quick fix!)
